.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,8 @@
+
+[//]: # (This should be discussed in the daily meeting)
+[//]: # (Remember to bump the version to the release version after this is merged)
```

Comment:
"after" or "before"? I guess this depends on the release guide being built, but I thought it would be before.